examples/cryptocurrency-advanced/backend/src/lib.rs

```diff
@@ -16,6 +16,7 @@
 
 #![deny(missing_debug_implementations, missing_docs, unsafe_code, bare_trait_objects)]
 
+extern crate exonum_crypto as crypto;
```

Comment:
I'd avoid renaming crates unless absolutely necessary (e.g., to avoid making gazillion of changes in existing codebase).

The crate _is_ called `exonum_crypto`, just as Testkit is called `exonum_testkit`. Plus we do have to fix the import list either way (as `crypto` is no longer a part of `exonum`), so we can simply use the proper name.